### PR TITLE
Issue #182 fix

### PR DESCRIPTION
**Pull request by MKiselev**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #182

### CODE REVIEW

File: generators/controller/Generator.php [hunk 1]
```diff
@@ -138,14 +138,19 @@ public function hints()
...
     {
         $actions = $this->getActionIDs();
-        if (in_array('index', $actions)) {
-            $route = $this->getControllerID() . '/index';
-        } else {
-            $route = $this->getControllerID() . '/' . reset($actions);
+
+        // https://github.com/yiisoft/yii2-gii/issues/182
+        $tryLink = '';
+        if (strpos($this->controllerNamespace, Yii::$app->controllerNamespace) === 0) {
```
Comment (samdark) [inline generators/controller/Generator.php]: I'd extract it into a separate method.

Comment (MKiselev) [inline generators/controller/Generator.php]: @samdark What exactly separate in to a method?
Check if controller namespace in sub-namespace of application or generation preview link?

Comment (samdark) [inline generators/controller/Generator.php]: Link generation.

Comment (MKiselev) [inline generators/controller/Generator.php]: @samdark okay, but most code was relocated from ```successMessage()``` to ```getLinkToTry()```

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**